istioctl/pkg/util/constants.go

```diff
@@ -18,6 +18,9 @@ const (
 	// DefaultProxyAdminPort is the default port for the proxy admin server
 	DefaultProxyAdminPort = 15000
 
+	// DefaultControlzPort is the default port for the ControlZ server
```

Comment:
It seems to be used even here - https://github.com/istio/istio/blob/ca8187a6b0efd36c958d38d0662a5d3ca1cb259e/pkg/ctrlz/options.go#L36

Reply:
I tried to modify this, but found that it is not allowed to import istioctl package here:
https://prow.istio.io/view/gs/istio-prow/pr-logs/pull/istio_istio/56334/lint_istio/1923689519576518656

Reply:
You can define a constant in pkg and import into istioctl pkg.

Reply:
updated, thx